src/Cdn/Cdn/Common/AzureCdnCmdletBase.cs

```diff
@@ -79,5 +81,12 @@ public void ConfirmAction(bool force, string actionMessage, Action action)
                 action();
             }
         }
+
+        protected override void BeginProcessing()
+        {
+            CdnMessageHelper.WriteMessage(this, 
+                "When you bump up Az.CDN module version, there will be changes to parameters and output, please use the new version: 2.0.0");
```

Comment:
Let's make the message close to using breaking change attributes
```suggestion
                $"Breaking changes in the cmdlet : {MyInvocation.Command.Name}
 - When you bump up Az.CDN module version, there will be changes to parameters and output, please use the new version: 2.0.0");
```
(Above code is not tested)

Reply:
Updated the message as required. the new display is shown in description section